File: app/src/main/java/com/stevenschoen/putionew/model/responses/FilesListResponse.java
package com.stevenschoen.putionew.model.responses;

import com.stevenschoen.putionew.model.files.PutioFile;

import java.util.List;

public class FilesListResponse extends BasePutioResponse {
	private List<PutioFile> files;
	private PutioFile parent;

	public List<PutioFile> getFiles() {
		return files;
	}

	public PutioFile getParent() { return parent; }
}

File: app/src/main/java/com/stevenschoen/putionew/model/responses/FilesListResponse.kt
package com.stevenschoen.putionew.model.responses

import com.stevenschoen.putionew.model.ResponseOrError
import com.stevenschoen.putionew.model.files.PutioFile

open class FilesListResponse(
    val files: List<PutioFile>,
    val parent: PutioFile
) : ResponseOrError.BasePutioResponse()
